Source/JavaScriptCore/wasm/WasmCalleeGroup.cpp

```diff
@@ -52,16 +52,20 @@ CalleeGroup::CalleeGroup(MemoryMode mode, const CalleeGroup& other)
     , m_llintCallees(other.m_llintCallees)
     , m_embedderCallees(other.m_embedderCallees)
     , m_wasmIndirectCallEntryPoints(other.m_wasmIndirectCallEntryPoints)
-    , m_wasmToWasmCallsites(other.m_wasmToWasmCallsites)
     , m_wasmToWasmExitStubs(other.m_wasmToWasmExitStubs)
+    , m_repatchTree(m_calleeCount)
 {
+    Locker locker { m_lock };
+    auto callsites = other.repatchTree().calleeGroupCallsites();
```

Comment:
Do we need a temporary variable here?

Reply:
Unfortunately, yes, to make WTFMove work.